security-admin/src/main/java/org/apache/ranger/biz/ServiceDBStore.java

```diff
@@ -4112,6 +4099,10 @@ private ServicePolicies getServicePolicies(String serviceName, Long lastKnownVer
             ret.setTagPolicies(tagPolicies);
         }
 
+        if (ret != null) {
```

Comment:
- replace `if (ret != null)` with inline check as shown below
- remove `<==` in the log message
```
LOG.debug("ServiceDBStore.getServicePolicies({}, {}): ret = {}", serviceName, lastKnownVersion, ret == null ? null : ret.getServiceConfig());
```